### PR TITLE
Homework17

### PR DESCRIPTION
**Pull request by LuckyCat26**

### CODE REVIEW

**Comment by Berichicko** [comment]
Please finish the Homework #17:TestNG Overview

1. Создайте новый файл Homework17.java с аннотацией @Test в IntelliJ IDEA.

2. Создайте тест кейс addSongToPlaylist() используя аннотацию @Test и хелперы, созданные ранее.

3. Перейдите на "https://bbb.testpro.io/".

4. Войдите с данными вашей учетной записи.

5. В поле поиска введите название песни (по Вашему выбору).

6. Нажмите View All.

7. Нажмите на первую песню в результатах поиска.

8. Нажмите на Add To... и выберите плейлист (Вы можете также создать новый плейлист).

9. Проверьте, что появился зеленый баннер с надписью Added 1 song into {your playlist}.

10. Создайте новую ветку и закоммитьте изменения.

11. Запуште код в удаленный репозиторий

12. Создайте пулл-реквест

💖Best wishes, the team HW - Alex!💖
E-mail: [aliaksandr@testpro.io](mailto:aliaksandr@testpro.io)

**Comment by LuckyCat26** [comment]
I already finished all the steps of the homework: commit 1ebab53 into main [17 hours ago]. Everything works on my end. I don't understand what's wrong. Could you, please, specify what exactly didn't work and what part I should correct?

**Comment by Berichicko** [comment]
> I already finished all the steps of the homework: commit [1ebab53](https://github.com/LuckyCat26/A44-QA-Automation-Framework/commit/1ebab53542e8624ab736e930f518e5dbee3ccec7) into main [17 hours ago]. Everything works on my end. I don't understand what's wrong. Could you, please, specify what exactly didn't work and what part I should correct?

Hello

Please, re-submit homework 17 for review

File: src/test/java/Homework17.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        clickViewAllBtn();
+        clickFirstSong();
+        waitCommand();
+        clickAddToBtn();
+        playlistNameInput("Practice");
+        clickSave();
+
+
+
+
```
Comment (Berichicko) [inline src/test/java/Homework17.java:25]: **Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._

**Comment by Berichicko** [comment]
Hello,

I have checked your homework 17, and I wanted to let you know that you're doing great! Keep up the excellent work!
Best wishes from the HW team (Alex, [aliaksandr@testpro.io](mailto:aliaksandr@testpro.io))!

804deb64dc406a20a75da05d91a1a69e

image

File: src/test/java/Homework17.java (hunk 1 above)
Comment (LuckyCat26) [reply src/test/java/Homework17.java:25]: Will do!!! Thanks for your feedback!

**Pull request merged**